test/powershell/Modules/Microsoft.PowerShell.Management/TimeZone.Tests.ps1

```diff
@@ -57,49 +56,53 @@ Describe "Get-Timezone test cases" -Tags "CI" {
     It "Call with ListAvailable switch returns a list containing TimeZoneInfo.Local" {
         $observedIdList = Get-TimeZone -ListAvailable | Select-Object -ExpandProperty Id
         $oneExpectedId = ([System.TimeZoneInfo]::Local).Id
-        $observedIdList -contains $oneExpectedId | Should Be $true
+        $observedIdList -eq $oneExpectedId | Should Be $oneExpectedId
     }
 
     It "Call with ListAvailable switch returns a list containing one returned by Get-TimeZone" {
         $observedIdList = Get-TimeZone -ListAvailable | Select-Object -ExpandProperty Id
         $oneExpectedId = (Get-TimeZone).Id
-        $observedIdList -contains $oneExpectedId | Should Be $true
+        $observedIdList -eq $oneExpectedId | Should Be $oneExpectedId
     }
 
     It "Call Get-TimeZone using ID param and single item" {
-        (Get-TimeZone -Id "Cape Verde Standard Time").Id -eq "Cape Verde Standard Time" | Should Be $true
+        $selectedTZ = $TimeZonesAvailable | Select-Object -Index ((Get-Random) % $TimeZonesAvailable.Count)
+        (Get-TimeZone -Id $selectedTZ.Id).Id | Should Be $selectedTZ.Id
```

Comment:
I think the tests should be deterministic and I would prefer don't use `Get-Random` because this reproducibly in manual locally _but not in CI restart_.
Perhaps it makes no sense to test all zones but it is very simple and very fast - I believe we may not worry about it here. (Although we may be limited to some selected time zone names/ids.)